.github/workflows/test.yml

```diff
@@ -1,42 +1,42 @@
-# name: Test
+name: Test
 
-# on: pull_request
+on: pull_request
 
-# jobs:
-#   test:
-#     name: Run linters and tests
-#     runs-on: ubuntu-latest
-#     steps:
-#       - name: Cancel Previous Runs
-#         uses: styfle/cancel-workflow-action@0.10.0
-#         with:
-#           access_token: ${{ github.token }}
+jobs:
+  test:
+    name: Run linters and tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
 
-#       - name: Checkout
-#         uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
 
-#       - name: Setup node and restore cached dependencies
-#         uses: actions/setup-node@v3
-#         with:
-#           node-version: '16'
-#           cache: 'npm'
+      - name: Setup node and restore cached dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```

Comment:
Regarding [this comment](https://github.com/manh-t/react-survey/pull/39#discussion_r1222909481), let's update this one as well

Reply:
@nvminhtue updated in [1c51466](https://github.com/manh-t/react-survey/commit/1c514664e38d1087e281b805600b731bf43c76c3)